commercetools-models/src/main/java/io/sphere/sdk/products/attributes/Attribute.java

```diff
@@ -36,16 +39,14 @@ static Attribute of(final String name, final JsonNode jsonNode) {
     }
 
     static <T> Attribute of(final NamedAttributeAccess<T> namedAttributeAccess, final T value) {
+
         final String name = namedAttributeAccess.getName();
-        //here is not the attributeMapper used to keep LocalizedEnum values which
-        //are transformed to just the key so the attribute could not be read anymore
-        final JsonNode jsonNode = SphereJsonUtils.toJsonNode(value);
-        if (value instanceof Reference && jsonNode instanceof ObjectNode) {
-            final Reference<?> reference = (Reference<?>) value;
-            if (reference.getObj() != null) {
-                ((ObjectNode) jsonNode).replace("obj", SphereJsonUtils.toJsonNode(reference.getObj()));
-            }
-        }
+        final ObjectMapper mapper = SphereJsonUtils.newObjectMapper();
```

Comment:
Here we could try to store the `ObjectMapper` in a static field - preferably in a package protected helper class.
This way we would avoid that we recreate it again and again. And as far as I can see this should be thread safe too. What do you think about it?

Reply:
sounds good to me too @katmatt i had also performance concern, but had no idea how to fix it 👍